test/functional/apps/management/_mgmt_import_saved_objects.js

```diff
@@ -10,21 +10,23 @@ import expect from '@kbn/expect';
 import path from 'path';
 
 export default function ({ getService, getPageObjects }) {
-  const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
   const PageObjects = getPageObjects(['common', 'settings', 'header', 'savedObjects']);
 
   //in 6.4.0 bug the Saved Search conflict would be resolved and get imported but the visualization
   //that referenced the saved search was not imported.( https://github.com/elastic/kibana/issues/22238)
 
   describe('mgmt saved objects', function describeIndexTests() {
-    beforeEach(async function () {
-      await esArchiver.emptyKibanaIndex();
-      await esArchiver.load('test/functional/fixtures/es_archiver/discover');
-      await PageObjects.settings.navigateTo();
+    before(async () => {
+      await kibanaServer.importExport.load('test/functional/fixtures/kbn_archiver/discover');
+    });
+
+    after(async () => {
+      await kibanaServer.importExport.unload('test/functional/fixtures/kbn_archiver/discover');
```

Comment:
I think we need the kbnArchiver clean() here.  This test imports saved objects from a file.  We should remove them in the after method.